### DISCUSSION TITLE
Opticks website down?

### DISCUSSION THREAD

**Discussion by chappedhands** [opticks-org/opticks#463] [Q&A] [answered]
Is the Opticks website down?

Is this the correct URL?
www.opticks.org

**Comment by tclarke** [comment, marked as answer]
It is working fine for me. We had some hosting provider maintenance earlier so you might have caught it during a short downtime.

**Reply by chappedhands** [reply]
It's working for me now as well.

**Reply by wkuperus** [reply]
A notification of server maintenance might be an idea as opposed to the site just being down? 
I can't access it at the moment (7th Feb '21 - 4:30 EST).

**Reply by tclarke** [reply]
It may actually be down. I haven't received any maintenance notices. I'll look into it.

**Reply by tclarke** [reply]
it's up now.

**Reply by kjkowale** [reply]
I think the Opticks.org website is no longer working?

**Reply by tclarke** [reply]
looks like it wants to update atlassian which is a manual process. I'll start it off and close this issue with a note when it's back up

**Reply by kjkowale** [reply]
ok, thanks.

**Reply by fshoemaker** [reply]
Website looks like it may be down again?
![Screenshot from 2022-06-09 12-34-13](https://user-images.githubusercontent.com/107206717/172898742-f47ec829-4bc6-4f08-a200-0e75e634d2b2.png)

**Comment by kjkowale** [comment]
Is Opticks web sire down again? Also, is the software being maintained since 2021?

**Comment by tclarke** [comment]
The server is having some issues.. It's blocking inbound and we're trying
to work out where. I think I'm going to try and redo a more simple version
on ghpages and just sunlight the historic data from the old site.
Is there something specific you are trying to access? I can make sure it's
available on GitHub if possible.

**Reply by fshoemaker** [reply]
@tclarke Thanks for the update. I was looking for the downloads page on opticks.org to provide to my company IT dept. I found the .msi I need here on GitHub. Thanks again for all you do!
